addons/mail/static/src/models/discuss_public_view/discuss_public_view.js

```diff
@@ -26,6 +26,7 @@ function factory(dependencies) {
                 }),
                 welcomeView: clear(),
             });
+            this.env.services.bus_service._restartPolling();
```

Comment:
I would move the line to `joinChannel` where it is more logical to be.
And add comment to explain the line is necessary for the poll to take into account the newly joined channel.